### ISSUE TITLE
validation accuracy

### ISSUE THREAD

**Issue by bzabdolahi** [Caenorst/YeNet-Pytorch#6]
Dear author
I'm using the BOSSbase_1.01 dataset, WOW algorithm 0.4bpp
4000 train, 1000 validation.
I train the model but I get  valid_accuracy=0.5
could you help me what went wrong?

**Comment by EmadHelmi** [comment]
I also have this problem. Could you fix that?
I also train on the BOSSbase with 10000 images and I put 3000 for validation but exactly I get the same result as you mentioned.
I would be happy If you help me.

**Comment by bzabdolahi** [comment]
my problem was solved by using "DatasetPair" class for data loading.

    On Wednesday, February 12, 2020, 11:41:22 AM GMT+3:30, EmadHelmi <notifications@github.com> wrote:  
 
 
I also have this problem. Could you fix that?
I also train on the BOSSbase with 10000 images and I put 3000 for validation but exactly I get the same result as you mentioned.
I would be happy If you help me.

**Comment by EmadHelmi** [comment]
Can I have your code, please?
Or can I send my code to you?

**Comment by bzabdolahi** [comment]
I didn't change the code. I just set 
batch_size = 32

test_batch_size = 32

lr = 4e-1

use_batch_norm = False

embed_otf = False

**Comment by TomTomTommi** [comment]
Sorry, I still not understand how to solve this problem, I try to use DatasetPair class for data loading, but the problem still here. coule you share the code, please?

**Comment by ByGary** [comment]
Does anyone sovle the above problem? The training loss and validation loss  always stay at a high level.
